src/main/java/nextstep/subway/path/dto/PathRequest.java

```diff
@@ -0,0 +1,14 @@
+package nextstep.subway.path.dto;
+
+import lombok.Getter;
+
+@Getter
+public class PathRequest {
+    private Long source;
+    private Long target;
```

Comment:
이 부분은 null이 되거나 하면 안될 것 같은데

`Bean Validation` 등을 활용해보시면 어떨까요?

```suggestion
    @NotNull
    private Long source;
    @NotNull
    private Long target;
```